be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MeetingValidator.scala

```diff
@@ -4,8 +4,8 @@ import ch.epfl.pop.model.network.JsonRpcRequest
 import ch.epfl.pop.model.network.method.message.Message
 import ch.epfl.pop.model.network.method.message.data.ObjectType
 import ch.epfl.pop.model.network.method.message.data.meeting.{CreateMeeting, StateMeeting}
-import ch.epfl.pop.model.objects.{Channel, Hash, PublicKey}
-import ch.epfl.pop.pubsub.graph.validators.MessageValidator._
+import ch.epfl.pop.model.objects.{Channel, Hash, PublicKey, WitnessSignaturePair}
+import ch.epfl.pop.pubsub.graph.validators.MessageValidator.{checkOwner, _}
```

Comment:
I guess it is better to simply keep the underscore, since `checkOwner` is already included into it.